src/Client.php

```diff
@@ -75,14 +75,21 @@ public function getUser(): User
      * @param SenderAddress|int|null $senderAddress The sender address to ship from. Methods available to all of your
      * account's sender addresses will be retrieved when null.
      * @param bool $returnMethodsOnly When true, methods for making a return are returned instead.
+     * @param int from_postal_code
+     * @param int to_postal_code
+     * @param int to_country iso2 code
      * @return ShippingMethod[]
      * @throws SendcloudClientException
      * @see https://sendcloud.dev/docs/shipping/shipping-methods/
      */
     public function getShippingMethods(
         ?int $servicePointId = null,
         SenderAddress|int|null $senderAddress = null,
-        bool $returnMethodsOnly = false
+        bool $returnMethodsOnly = false,
+        ?int $from_postal_code =null,
+        ?int $to_postal_code =null,
+        ?string $to_country =null,
+        
```

Comment:
Empty line:

```suggestion
```

And: To bring this fully in line with the documentation all that's needed still is the `is_return` boolean argument. Let's add that while we're at it?